src/Microsoft.IdentityModel.Tokens/Telemetry/TelemetryDataRecorder.cs

```diff
@@ -26,9 +26,16 @@ internal class TelemetryDataRecorder
         /// <summary>
         /// Counter to capture configuration refresh requests to ConfigurationManager.
         /// </summary>
+        internal static readonly Counter<long> ConfigurationManagerCounter = IdentityModelMeter.CreateCounter<long>(IdentityModelConfigurationManagerCounterName, description: IdentityModelConfigurationManagerCounterDescription);
         internal const string IdentityModelConfigurationManagerCounterName = "IdentityModelConfigurationManager";
         internal const string IdentityModelConfigurationManagerCounterDescription = "Counter capturing configuration manager operations.";
-        internal static readonly Counter<long> ConfigurationManagerCounter = IdentityModelMeter.CreateCounter<long>(IdentityModelConfigurationManagerCounterName, description: IdentityModelConfigurationManagerCounterDescription);
+
+        /// <summary>
+        /// Counter to capture background refresh failures in the ConfigurationManager.
+        /// </summary>
+        internal static readonly Counter<long> BackgroundRefreshFailureCounter = IdentityModelMeter.CreateCounter<long>(BackgroundRefreshFailureCounterName, description: BackgroundRefreshFailureCounterDescription);
+        internal const string BackgroundRefreshFailureCounterName = "IdentityModelConfigurationManagerBackgroundRefreshFailure";
+        internal const string BackgroundRefreshFailureCounterDescription = "Counter capturing configuration manager background refresh failures.";
```

Comment:
I think it'd be clearer to specify what the refresh refers to.
```suggestion
        internal static readonly Counter<long> BackgroundConfigurationRefreshFailureCounter = IdentityModelMeter.CreateCounter<long>(BackgroundConfigurationRefreshFailureCounterName, description: BackgroundConfigurationRefreshFailureCounterDescription);
        internal const string BackgroundConfigurationRefreshFailureCounterName = "IdentityModelConfigurationManagerBackgroundRefreshFailure";
        internal const string BackgroundConfigurationRefreshFailureCounterDescription = "Counter capturing configuration manager background refresh failures.";
```